### ISSUE TITLE
rasa_nlu.evaluate.run_cv_evaluation provides precision only for the last fold tried, so you can't take average precision across folds

### ISSUE THREAD

**Issue by fabigato** [RasaHQ/rasa_nlu#833]
**Rasa NLU version** (e.g. `0.11.2`): 0.11.2

**Used backend / pipeline** (`mitie`, `spacy_sklearn`, ...): spacy_sklearn

**Operating system** (windows, osx, ...): ubuntu 16.04

**Issue**:
rasa_nlu.evaluate.run_cv_evaluation saves accuracy and f1 scores for each fold, so later you can check the average across folds, but for precision metric, it doesn't keep it for all folds, only for the current one (overwriting the previous). These are the relevant code lines (in rasa_nlu.evaluate, line 456 onwards):
```
        results["Accuracy"].append(metrics.accuracy_score(test_y, preds))
        results["F1-score"].append(metrics.f1_score(test_y, preds, average='weighted'))
        results["Precision"] = metrics.precision_score(test_y, preds, average='weighted')
```
Shouldn't the last line use append also, instead of = ?

**Comment by tmbo** [comment]
Yes that seems to be off - mind providing a PR to fix this?

**Comment by fabigato** [comment]
Should I create a fork and merge from it? I didn't find the right procedure in the docs (https://rasahq.github.io/rasa_nlu/contribute.html). Which head branch should I use when creating the PR?

**Comment by fabigato** [comment]
nevermind, I see the fix now in 3d8d70c 2 hours ago.

**Comment by tmbo** [comment]
:+1:
